datasets/newsqa/newsqa.py

```diff
@@ -0,0 +1,326 @@
+# coding=utf-8
+# Copyright 2020 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This code is used to read and load NewsQA dataset."""
+
+from __future__ import absolute_import, division, print_function
+
+import csv
+import json
+import os
+
+import datasets
+
+
+# Find for instance the citation on arxiv or on the dataset repo/website
+_CITATION = """\
+@inproceedings{trischler2017newsqa,
+  title={NewsQA: A Machine Comprehension Dataset},
+  author={Trischler, Adam and Wang, Tong and Yuan, Xingdi and Harris, Justin and Sordoni, Alessandro and Bachman, Philip and Suleman, Kaheer},
+  booktitle={Proceedings of the 2nd Workshop on Representation Learning for NLP},
+  pages={191--200},
+  year={2017}
+}
+
+"""
+
+# You can copy an official description
+_DESCRIPTION = """\
+NewsQA is a challenging machine comprehension dataset of over 100,000 human-generated question-answer pairs. \
+Crowdworkers supply questions and answers based on a set of over 10,000 news articles from CNN, with answers consisting of spans of text from the corresponding articles.
+"""
+
+_HOMEPAGE = "https://www.microsoft.com/en-us/research/project/newsqa-dataset/"
+
+_LICENSE = 'NewsQA Code\
+Copyright (c) Microsoft Corporation\
+All rights reserved.\
+MIT License\
+Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:\
+The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.\
+THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.\
+© 2020 GitHub, Inc.'
+
+
+class NewsQA(datasets.GeneratorBasedBuilder):
```

Comment:
```suggestion
class Newsqa(datasets.GeneratorBasedBuilder):
```
This must be the camel case version of the dataset script name `newsqa`

You'll need to regenerate the dataset_infos.json file after this change

Reply:
The change is only the class name right? 
`NewsQA ` to `Newsqa ` just one ? 
I've done this in the latest commit.

Reply:
Yep exactly, thank you :)